### PR TITLE
Update Individual chemical page

### PR DESCRIPTION
**Pull request by ngu892**
Individual chemical page is ready to show up,
the Path is
http://localhost:3000/chemicals/{chemical.id}

db.json is updated with yao's  json file

chemicals image names are changed to id number 

delete unnecessary chemical resources
- chemical-fetchers
- chemical-grid
- chemical-products
- chemical-types

Pls check the code and review it 

Thx

### CODE REVIEW

File: next-app/src/app/(app-layout)/chemicals/[chemicalId]/page.tsx [hunk 1]
```diff
@@ -1,16 +1,35 @@
-import { Chemical } from '@/common/object-types';
+import { Id2 } from '@/common/common-types';
```
Comment (mattbeard03) [inline next-app/src/app/(app-layout)/chemicals/[chemicalId]/page.tsx:1]: Id not Id2? Is this a new type?

File: next-app/src/app/(app-layout)/flavours/[flavourId]/page.tsx [hunk 2]
```diff
@@ -1,55 +1,53 @@
...
-import Image from 'next/image';
-import type { Flavour } from '@/common/object-types';
+import { Id } from '@/common/common-types';
 import { PageTitle } from '@/common/page-title';
 import { Paper } from '@/common/paper';
 import { Section, SectionTitle } from '@/common/section';
 import { getMetadata } from '@/seo/seo-utils';
-import { RelatedProducts } from '@/whiskeys/related-whiskey';
+
+import { RelatedProducts, RelatedProductType } from '@/whiskeys/related-whiskey';
```
Comment (mattbeard03) [inline next-app/src/app/(app-layout)/flavours/[flavourId]/page.tsx:8]: Note to @mattbeard03 to change this to "RelatedWhiskey"

File: next-app/src/common/common-types.ts [hunk 3]
```diff
@@ -4,4 +4,6 @@ export type Maybe<T> = T | Nil;
 
 export type Id = number;
 
+export type Id2 = number;
```
Comment (mattbeard03) [inline next-app/src/common/common-types.ts:7]: why not just use Id ?

**Review by mattbeard03 — APPROVED** [review]
Looks good, just need clarification on the Id2 vs Id. Functionally fine, but would be good to condense to one.

File: next-app/src/common/common-types.ts (hunk 3 above)
Comment (ngu892) [reply next-app/src/common/common-types.ts:7]: When I tried to fix the routing error, I have changed Id > Id2, I can re change it to Id

Comment (mattbeard03) [reply next-app/src/common/common-types.ts:7]: > When I tried to fix the routing error, I have changed Id > Id2, I can re change it to Id

No need, understood, I will get it during my weekly file clean up.

**Pull request merged**